docgen.mjs

```diff
@@ -49,8 +49,6 @@ function trimApostrophes(str) {
     return str;
 }
 
-const components = Object.keys(componentObj); // Replace with your actual component names
```

Comment:
Don't believe this was actually doing anything. Lint was flagging as unused declaration